desktop/renderer-app/src/pages/HomePage/index.tsx

```diff
@@ -29,6 +31,18 @@ export const HomePage = observer<HomePageProps>(function HomePage() {
     }, [lastLocation]);
 
     useEffect(() => {
+        const checkUpdateVisible =
+            formatDistanceStrict(globalStore.checkNewVersionDate, new Date().getTime(), {
+                unit: "hour",
+            }) === "12 hours";
```

Comment:
```suggestion
            differenceInHours(new Date().getTime(), globalStore.checkNewVersionDate) >= 12;
```

Reply:
@BlackHole1 Because we don't want to send a request to check for updates every time we click on the Homepage component.